collect_app/src/main/java/org/odk/collect/android/backgroundwork/BackgroundWorkManager.java

```diff
@@ -0,0 +1,12 @@
+package org.odk.collect.android.backgroundwork;
+
+public interface BackgroundWorkManager {
+
+    boolean isFormUploaderRunning();
```

Comment:
Feels weird that this doesn't also schedule and cancel form upload and download but I assume it'll come.

Reply:
Yeah it's on the way!